tests/fixtures/yaml_tests/failing_test_variable_relative_error_margin.yaml

```diff
@@ -0,0 +1,9 @@
+- name: "Failing test: result out of variable specific relative error margin"
+  period: 2015-01
+  relative_error_margin:
+    default: 1
```

Comment:
I don't understand here the logic. Relative error margin to relative to what? Are we here again just overriding the relative margin error just fro income tax? 

Maybe an example where this is useful including other variables could come handy to better grasp the intent.

Reply:
relative error margin = abs(computed_value - target_value / target_value)  is here 100%.

An example is pension there are some variables where it is very difficult to compute up to 1% error.
And other where it is crucial.